app.py

```diff
@@ -4,13 +4,29 @@
 #create an instance of Flask class
 app = Flask(__name__)
 
-@app.route('/')
+@app.route('/lynn')
+def login():
+    return render_template('login.html')
+
+@app.route('/lynn')
 def index():
     return render_template('index.html')
 
-@app.route('/user/<name>')
-def user(name):
-    return render_template('user.html', name=name)
+@app.route('/lynn')
+def lynn():
+    return render_template('lynn.html')
+
+@app.route('/lynn')
```

Comment:
This won't work when we run the app. A route is always supposed to be unique. If we render multiple templates on the same route (/lynn here ) flask won't be able to resolve it. It's a one to many mapping.

Reply:
So in this case for making it work we can rename in to /friends or anything relevant,